modules/appeals_api/app/workers/appeals_api/higher_level_review_upload_status_batch.rb

```diff
@@ -6,7 +6,8 @@ module AppealsApi
   class HigherLevelReviewUploadStatusBatch
     include Sidekiq::Worker
 
-    sidekiq_options 'retry': true, unique_until: :success
+    # No need to retry since the schedule will run this every hour
+    sidekiq_options retry: false, unique_until: :success
```

Comment:
is the `unique_until` necessary if it's `false`?

Reply:
Fix'd :)